hikari/impl/rest.py

```diff
@@ -1137,14 +1137,27 @@ async def delete_message(
     async def delete_messages(
         self,
         channel: snowflakes.SnowflakeishOr[channels_.TextChannel],
+        messages: typing.Union[
+            snowflakes.SnowflakeishOr[messages_.PartialMessage],
+            snowflakes.SnowflakeishIterable[messages_.PartialMessage],
+        ],
         /,
-        *messages: snowflakes.SnowflakeishOr[messages_.PartialMessage],
+        *other_messages: snowflakes.SnowflakeishOr[messages_.PartialMessage],
     ) -> None:
         route = routes.POST_DELETE_CHANNEL_MESSAGES_BULK.compile(channel=channel)
 
-        pending: typing.Deque[snowflakes.SnowflakeishOr[messages_.PartialMessage]] = collections.deque(messages)
+        pending: typing.Deque[snowflakes.SnowflakeishOr[messages_.PartialMessage]] = collections.deque()
         deleted: typing.Deque[snowflakes.SnowflakeishOr[messages_.PartialMessage]] = collections.deque()
 
+        if isinstance(messages, typing.Iterable):  # Syntactic sugar. Allows to use iterables
```

Comment:
Just for consistency, could you move the comment above the line?

Then feel free to merge :)

Reply:
Does this mean that other places where short comments aren't put on their own line also be refactored for consistency at one point?

Reply:
This isnt a comment on that specific line, this is a comment about the block that is under it, which is why it should be above

Reply:
hmm why above? isn't it generally kept within the if statement in this module since that's where the block starts